layouts/shortcodes/fileRel.html

```diff
@@ -0,0 +1,6 @@
+{{- $pathURL := .Get "pathURL" -}}
+{{- $name := .Get "name" -}}
+{{- $class := .Get "class" -}}
+{{- $style := .Get "style" -}}
+
+<a class="{{ $class }}" style="{{ $style | safeCSS }}" href="{{ $pathURL | relURL }}">{{$name}}</a>
```

Comment:
```suggestion
<a class="{{ $class }}" style="{{ $style | safeCSS }}" href="{{ $pathURL | relURL }}">{{ $name }}</a>
```

add inner spacing to double curly brackets around `$name` to be consistent